_translator-files/po/de/Installation-for-Linux.po

```diff
@@ -42,7 +43,7 @@ msgstr "Installation auf Linux"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr "{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte\" branch1-url=\"Getting-Started\" %}"
+msgstr "{% include breadcrumb.html root=\"Jamulus verwenden \" branch1=\"Setup - Erste Schritte mit Jamulus\" branch1-url=\"Getting-Started\" %}"
```

Comment:
Please be consistent with macOS and only say Erste Schritte here. Same for all other OS